db/schema.rb

```diff
@@ -409,6 +346,54 @@
     GROUP BY rejected.prison_name, rejected.reason, round((((rejected.rejected_count)::numeric / (booked.booked_count)::numeric) * (100)::numeric), 2), (date_part('year'::text, rejected.created_at))::integer, (date_part('month'::text, rejected.created_at))::integer, (date_part('day'::text, rejected.created_at))::integer;
   SQL
 
+  create_view :timely_and_overdue_by_calendar_weeks,  sql_definition: <<-SQL
```

Comment:
I wish it would keep the same order.  I manually diffed it to fix some PG optimisations that only seem to happen on my machine around the distribution views.  I'm afraid that trying to get the move of this `create_view` diffed out manually is a bit too high-risk for my liking.